resources/css/ceres-item.scss

```diff
@@ -14949,2932 +14830,2914 @@ $border-width: 1px;
-				padding: 0.5rem .8rem;
-				display: inline-block;
-				color: $gray-lighter;
-
-				&:hover {
-					@include context-bg($primary);
-				}
-			}
-
-			// icons should be slightly larger
-			.fa:not(.qty-sign){
-				font-size: 1.2rem;
-			}
-
-			.basket-seperator{
-				margin: 0 2px;
-			}
-		}
-
-		.control-user{
-			-webkit-box-flex: 1;
-			    -ms-flex: 1 0 auto;
-			        flex: 1 0 auto;
-			//login | register
-			#login-change {
-				position:relative;
-
-				& > .dropdown {
-
-					& .dropdown-toggle {
-						-webkit-transition: all .1s;
-						transition: all .1s;
-	
-						&:after {
-							margin-left: 7px;
-						}
-					}
-	
-					&.show {
-						& > .dropdown-toggle {
-							@include context-bg($gray-darker);
-						}
-	
-						& .dropdown-menu {
-							max-height: 200px;
-
-							@include media-breakpoint-down(md) {
-								width: 105px;
-
-								&.dropdown-menu-right {
-									right: initial;
-								}
-							}
-						}
-					}
-	
-					.dropdown-menu {
-						z-index: 2000;
-						-webkit-transition: all 0.3s;
-						transition: all 0.3s;
-						max-height: 0;
-						display: block;
-						overflow: hidden;
-						border: none !important;
-
-						@include media-breakpoint-down(md) {
-							width: 105px;
-
-							&.dropdown-menu-right {
-								right: initial;
-							}
-						}
-
-						a {
-							padding:1rem .8rem;
-							border:0;
-							border-top: 1px solid $gray-lighter;
-							display: block;
-							background-color: $gray-darker;
-							color: color-yiq($gray-darker);
-							&:hover {
-								color: color-yiq($primary);
-								background-color: $primary;
-							}
-	
-							& > .fa-user {
-								margin-right: 5px;
-							}
-	
-							& > .fa-sign-out {
-								margin-right: 3px;
-							}
-						}
-					}
-				}
-			}
-		}
-
-		.control-wish-list {
-			a {
-				.badge-right{
-					margin-right: .5rem; 
-				}
-				i{
-					&.fa-heart-o{
-						font-size: .9rem;
-					}
-
-					&.fa-heart{
-						margin-right: .2rem;
-						font-size: .8rem;
-						color: rgba(255, 0, 0, 0.67);
-					}
-				}
-			}
-		}
-
-		.control-languages a[aria-expanded="true"],
-		.control-search a[aria-expanded="true"] {
-			background:$gray-darker;
-
-			& > i.fa.fa-search {
-				display: none;
-			}
-
-			& > i.fa.fa-close {
-				display: inherit;
-			}
-		}
-
-		.control-search a {
-
-			& > i {
-				min-width: 16px;
-			}
-			
-			& > i.fa.fa-search {
-				display: inherit;
-			}
-
-			& > i.fa.fa-close {
-				display: none;
-			}
-		}
-
-		.control-basket{
-			position: relative;
-
-			.toggle-basket-preview{
-				background-color: $top-bar-basket-btn;
-				color: $top-bar-basket-color;
-
-				.badge{
-					font-size: inherit;
-					padding: 0;
-					line-height: inherit;
-					font-weight: inherit;
-					margin-left: .5rem;
-				}
-
-				.badge-right{
-					margin-right: .5rem; 
-				}
-
-				&:hover{
-					background-color: darken($top-bar-basket-btn, 10%);
-				}
-			}
-		}
-	}
-}
+.top-bar {
```

Comment:
Dieser Code liegt nun in ceres-category.scss, ceres-checkout.scss und ceres-item.scss. Mir ist auf den 1. Blick kein unterschied aufgefallen, wie fändest du es hierfür ein mixin zu erstellen, welches in den einzelnen Dateien einbunden wird? So muss man bei Änderungen nicht mehrere Stellen anpassen.

Reply:
Das ist korrekt und so gewollt. hierbei handelt es sich um die autom. generierte scss-Datei an

Reply:
👍